src/react-native-animated-charts/src/charts/linear/ChartDot.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { ViewProps } from 'react-native';
+import Animated, {
+  useAnimatedStyle,
+  withSpring,
+} from 'react-native-reanimated';
+import { useChartData } from '../../helpers/useChartData';
+import withReanimatedFallback from '../../helpers/withReanimatedFallback';
+import { FIX_CLIPPED_PATH_MAGIC_NUMBER } from './ChartPath';
+
+interface ChartDotProps extends ViewProps {
+  size?: number;
+  springConfig?: Animated.WithSpringConfig;
+}
+
+const springDefaultConfig = {
+  damping: 15,
+  mass: 1,
+  stiffness: 600,
+};
+
+const ChartDot: React.FC<ChartDotProps> = ({
+  style,
+  size = 10,
+  springConfig,
+  ...props
+}) => {
```

Comment:
I think we're trying to use `React.FC`? I think this is the same as this, no?

```suggestion
const ChartDot = ({
  style,
  size = 10,
  springConfig,
  ...props
} : ChartDotProps) => {
```

Reply:
No, React.FC also tells you that the function should return a valid React element as well as adds types for a children prop.